src/components/button-toggle/button-toggle.style.ts

```diff
@@ -65,6 +67,40 @@ const StyledButtonToggle = styled.button<StyledButtonToggleProps>`
   box-sizing: border-box;
   max-width: 100%;
 
+  ${({ variant }) =>
+    variant === "minor" &&
+    css`
+      border: 1px solid var(--colorsActionMinor500);
+
+      &&&&&& {
+        border-radius: var(--borderRadius050);
+        padding: var(--spacing000) var(--spacing100) var(--spacing000)
+          var(--spacing100);
+      }
+
+      &:focus {
```

Comment:
Comment: you don't need this as far as I can tell. You should style it using the `aria-pressed=true` attribute similar to how it is achieved above. You'll also need to the padding imo as when the toggle button is active the increased border thickness will cause the layout to jump so the padding will need to be reduced when that happens to avoid it. Something like the below will work:

```
color: var(--colorsActionMinor500);
      border: 1px solid var(--colorsActionMinor500);

      &[aria-pressed="true"] {
        border: 4px solid var(--colorsActionMinor500);
        background-color: transparent;
        padding: 0 ${paddingConfig[size] - 3}px;
        
        :hover {
          background-color: var(--colorsActionMinor600);
          color: var(--colorsUtilityYang100);
          border-color: var(--colorsActionMinor600);
        }
      }

      :not([aria-pressed="true"]):not(:disabled):hover {
        color: var(--colorsActionMinorYang100);
        background-color: var(--colorsActionMinor600);
        border-color: var(--colorsActionMinor600);
      }
```

Reply:
https://codesandbox.io/s/carbon-quickstart-forked-f82m7p?file=/src/App.js

If you click one of the buttons you will see the content jumping